tracing/src/main/java/io/airlift/tracing/OpenTelemetryConfig.java

```diff
@@ -22,4 +24,17 @@ public OpenTelemetryConfig setEndpoint(String endpoint)
         this.endpoint = endpoint;
         return this;
     }
+
+    @FileExists
+    public String getResourceAttributesFile()
+    {
+        return resourceAttributesFile;
+    }
+
+    @Config("tracing.resource-attributes-file")
```

Comment:
We should just have a common name, we are calling it different things in the separate modules name with the same intention, and we could use a common property name so we just set it once in config. This looks good otherwise.